### PR TITLE
D8: update to beta13, YAML make file, and incorporate variables into $settings

### PR DESCRIPTION
**Pull request by pjcdawkins**

### CODE REVIEW

File: settings.php [hunk 1]
```diff
@@ -7,9 +7,6 @@
 // Install with the 'standard' profile for this example.
 $settings['install_profile'] = 'standard';
 
-// You should modify the hash_salt so that it is specific to your application.
-$settings['hash_salt'] = '4946c1912834b8477cc70af309a2c30dcec24c2103c724ff30bf13b4c10efd82';
-
```
Comment (damz) [inline settings.php]: This is still needed by D8, no?

Comment (pjcdawkins) [inline settings.php]: @damz yes, but it's a key in $settings, so it can be added in platform variables

Comment (damz) [inline settings.php]: It can, but it won't if you initialize the project through the UI. We really should not add an additional step here :)

**Comment by OriPekelman** [comment]
Any reason not to go to -beta14 ?

**Comment by pjcdawkins** [comment]
seems to be superseded by #49

**Pull request closed without merging**